assets/elasticsearch/Dockerfile

```
@@ -1,21 +1,24 @@
+    elasticsearch=8.6.2 \
     libcurl4 \
     libcurl4-openssl-dev \
-    elasticsearch=8.6.2 \
-    python3-pip
+    python3-pip \
+    python3-venv \
+    tmux \
+    vim;
 
-RUN pip install elasticsearch requests numpy matplotlib
+RUN pip3 install elasticsearch==8.6.2 requests;
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)